python/ray/tests/test_metrics.py

```diff
@@ -112,8 +112,9 @@ def f(self):
         ]
         for process in processes:
             # TODO(ekl) why does travis/mi end up in the process list
-            assert ("python" in process or "ray" in process
-                    or "travis" in process)
+            assert ("python" in process or "conda" in process
+                    or "travis" in process or "runner" in process
```

Comment:
runner? why?

Reply:
I think `travis` was meant to refer to the username on Travis, so I added `runner`, which is the username on GitHub Actions.